client/lib/domains/resolve-domain-status.tsx

```diff
@@ -468,28 +469,51 @@ export function resolveDomainStatus(
 					icon: 'check_circle',
 				};
 			}
+			// domain.transferStatus === transferStatus.COMPLETED &&
+			if ( ! domain.pointsToWpcom ) {
```

Comment:
Just a note to fix this conditional before merging. ☺️

Reply:
Oh-no!!! Fixed 🤦